config/core/300-resources/route.yaml

```diff
@@ -23,37 +25,149 @@ metadata:
 spec:
   group: serving.knative.dev
   versions:
-  - name: v1
```

Comment:
was the indentation incorrect before then?

Reply:
Nope, just a different style. The tool just does that though.

Reply:
🤷 fair enough